FileProcessor/src/main/java/com/group7/FileProcessor/ScenarioMaker.java

```diff
@@ -8,61 +8,45 @@
 import com.group7.FileProcessor.pojo.*;
 import java.io.IOException;
 
-
 /**
  *
  * @author keric
  */
```

Comment:
If this class is obsolete now I would recommend adding an @Deprecated annotation right here on line 15 before the class starts.